libs/perun/components/src/lib/resources-list/resources-list.component.ts

```diff
@@ -93,6 +93,9 @@ export class ResourcesListComponent implements AfterViewInit, OnChanges {
   }
 
   ngAfterViewInit(): void {
+    if (window.location.href.indexOf("/organization") === -1){
```

Comment:
Although this implementation might work, it is not a good way of doing this. A better way is to define a new input property called `hideColumns` and use this to specify, which columns should be hidden. Then on a place where this component is used and you want to hide some column, you can pass to this property the desired value.

This is already implemented in some of our components. For example, you can take a look at the `GroupsListComponent`.